Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -145,6 +159,22 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   [self sendEventWithName:@"remoteNotificationsRegistered" body:notification.userInfo];
 }
 
+- (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
+{
+  if (self.requestPermissionsResolveBlock == nil) {
+    return;
+  }
+  UIUserNotificationSettings *notificationSettings = notification.userInfo;
+  NSDictionary *notificationTypes = @{
+    @"alert": @((BOOL)notificationSettings.types & UIUserNotificationTypeAlert),
```

Comment:
I think this needs to be written as

     @"alert": @(notificationSettings.types & UIUserNotificationTypeAlert > 0)